src/frontend/csrs-portal/src/app/components/questionnaire/questionnaire.component.ts

```diff
@@ -715,37 +689,15 @@ export class QuestionnaireComponent implements OnInit {
   }
 
   register() {
```

Comment:
this empty function not doing anything can disappear too - right?

Reply:
it was removed